### PR TITLE
Fix problem externals does not get included

### PR DESCRIPTION
**Pull request by titipata**
This was my mistake when adding `externals` folder. We have to fix the `setup.py` too.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,7 +67,7 @@ def get_version():
             "Programming Language :: Python :: 3.7",
         ],
         platforms="any",
-        packages=["pyglmnet"],
+        packages=package_tree("pyglmnet"),
```
Comment (jasmainak) [inline setup.py]: why not:

```py
from setuptools import find_packages
```

and then here:

```suggestion
        packages=find_packages(),
```

Comment (titipata) [reply setup.py]: I was gonna do that, not sure which way is good. Will come fix it after lunch!

**Comment by titipata** [comment]
@jasmainak resolve your comments!

**Comment by jasmainak** [comment]
I’ll merge once CIs are green :)

**Comment by jasmainak** [comment]
Thanks @titipata !

**Pull request merged**